### PR TITLE
Fix syntax error in invoke command

### PR DESCRIPTION
**Pull request by sblumenthal**
### What does this PR do?

Fixes a syntax error introduced in https://github.com/DataDog/datadog-agent/pull/19784

### Motivation

After the change introduced in the linked PR, the CI job was failing with:
```
No idea what 'false' is!
```

The correct syntax for setting a boolean invoke parameter is `--no-<name>`, not `--<name> False`

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by chouetz — APPROVED** [review]

File: .gitlab/internal_kubernetes_deploy.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@ internal_kubernetes_deploy_experimental:
   script:
     - source /root/.bashrc
     - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_pipelines_scheduler_token --with-decryption --query "Parameter.Value" --out text)
-    - inv pipeline.trigger-child-pipeline --project-name "DataDog/k8s-datadog-agent-ops" --git-ref "main" --variables "OPTION_AUTOMATIC_ROLLOUT,EXPLICIT_WORKFLOWS,OPTION_PRE_SCRIPT,FILTER,SKIP_PLAN_CHECK" --follow false
```
Comment (chouquette) [inline .gitlab/internal_kubernetes_deploy.yml:31]: Is the `FILTER` removal intentional? It doesn't seem related to the described change

Comment (sblumenthal) [reply .gitlab/internal_kubernetes_deploy.yml:31]: Yea, the variable was removed in the linked PR, but I did not see it in the passed in variable list on this line.

**Review by davidor — APPROVED** [review]

File: .gitlab/internal_kubernetes_deploy.yml (hunk 1 above)
Comment (chouquette) [reply .gitlab/internal_kubernetes_deploy.yml:31]: Thanks for clearing it out :)

**Review by chouquette — APPROVED** [review]

**Review by L3n41c — APPROVED** [review]
